ciao-cli/instance.go

```diff
@@ -355,18 +358,34 @@ func (cmd *instanceListCommand) run(args []string) error {
 		fatalf(err.Error())
 	}
 
+	w := new(tabwriter.Writer)
+	if !cmd.detail {
+		w.Init(os.Stdout, 0, 1, 1, ' ', 0)
+		fmt.Fprintln(w, "#\tUUID\tStatus\tPrivate IP\tSSH IP\tSSH PORT")
+	}
+
 	for i, server := range servers.Servers {
-		fmt.Printf("Instance #%d\n", i+1)
-		fmt.Printf("\tUUID: %s\n", server.ID)
-		fmt.Printf("\tStatus: %s\n", server.Status)
-		fmt.Printf("\tPrivate IP: %s\n", server.Addresses.Private[0].Addr)
-		fmt.Printf("\tMAC Address: %s\n", server.Addresses.Private[0].OSEXTIPSMACMacAddr)
-		fmt.Printf("\tCN UUID: %s\n", server.HostID)
-		fmt.Printf("\tImage UUID: %s\n", server.Image.ID)
-		fmt.Printf("\tTenant UUID: %s\n", server.TenantID)
-		if server.SSHIP != "" {
-			fmt.Printf("\tSSH IP: %s\n", server.SSHIP)
-			fmt.Printf("\tSSH Port: %d\n", server.SSHPort)
+		if !cmd.detail {
+			fmt.Fprintf(w, "%d", i+1)
+			fmt.Fprintf(w, "\t%s", server.ID)
+			fmt.Fprintf(w, "\t%s", server.Status)
+			fmt.Fprintf(w, "\t%s", server.Addresses.Private[0].Addr)
+			fmt.Fprintf(w, "\t%s", server.SSHIP)
```

Comment:
server.SSHIP can be an empty string in which case this is a property that we do not export. Thus I'd like those 2 lines to be handled this way:
```
			if server.SSHIP != "" {
				fmt.Fprintf(w, "\t%s", server.SSHIP)
				fmt.Fprintf(w, "\t%d", server.SSHPort)
			} else {
				fmt.Fprintf(w, "\tN/A")
				fmt.Fprintf(w, "\tN/A")
			}
```